public/js/public/login.js

```diff
@@ -44,19 +34,60 @@ $(document).ready(function () {
         password: $('.login-form input[type="password"]').val()
       },
       success: function (response) {
-        $('.login-form').html(Lang.get('user/login.login_success'));
-        $('.login-form').css('color', 'green');
-        localStorage.setItem('token-login', response.result.token);
-        localStorage.setItem('username', response.result.user.username);
-        $('#userName').html(response.result.user.username);
-        window.location.href = 'http://' + window.location.hostname;
+        loginSuccess(response);
       },
       error: function (response) {
         alert(response.responseJSON.error);
         $('.login-form input[type="password"]').val('');
       }
     });
   })
-  name = localStorage.getItem("username");
-  $('#userName').html(name);
 })
+function onSignIn(googleUser) {
```

Comment:
googleUser ở đâu ra em?

Reply:
https://developers.google.com/identity/sign-in/web/sign-in, em làm theo cái link ni đây ạ